src/ccl/ccl.module.ts

```diff
@@ -0,0 +1,36 @@
+import { NgModule } from '@angular/core';
```

Comment:
Looks like there was maybe a bad merge conflict resolution here. This file should no longer exist, any modifications you made to it should be moved to `app.module.ts`